net/remote/remote-node.sh

```diff
@@ -263,7 +263,7 @@ EOF
       agave-ledger-tool -l config/bootstrap-validator shred-version --max-genesis-archive-unpacked-size 1073741824 | tee config/shred-version
 
       if [[ -n "$maybeWaitForSupermajority" ]]; then
-        bankHash=$(agave-ledger-tool -l config/bootstrap-validator bank-hash --halt-at-slot 0)
+        bankHash=$(agave-ledger-tool -l config/bootstrap-validator verify --halt-at-slot 0 --print-bank-hash | jq -r .hash)
```

Comment:
Need to specify '--output json' here, still doesn't work, I'll take another look after the meetings.

Reply:
🤦 - https://github.com/anza-xyz/agave/pull/1745/commits/ca7a098e22d1e7cf2e7ce8647a59b3b97651f4a3

Reply:
Okay, this worked for me:
bankHash=$(agave-ledger-tool -l config/bootstrap-validator verify --halt-at-slot 0 --print-bank-hash --output json | jq -r ".hash")